base_report_info/models/res_company.py

```diff
@@ -0,0 +1,12 @@
+from odoo import fields, models
+
+
+class Company(models.Model):
+    _inherit = "res.company"
+
+    bank_info = fields.Char(
+        name="Bank",
+    )
+    description = fields.Char(
+        name="Description",
+    )
```

Comment:
What description is it? Make the field name more specific so it better represents the purpose.